src/components/attendee/Banner/Banner.module.scss

```diff
@@ -0,0 +1,42 @@
+@use "scss/attendee/layer";
+@use "scss/attendee/theme";
+@use "scss/attendee/space";
+
+$-z-index--banner: layer.above(0);
+
+.banner {
+  position: fixed;
+  z-index: $-z-index--banner;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  opacity: 1;
+
+  transition: opacity 1.5s ease;
+  transition-delay: 1s;
```

Comment:
[NOTE] If we see more of these transitions, we may extract them behind few constants for reuse everywhere.